openedx_tagging/core/tagging/import_export/actions.py

```diff
@@ -256,16 +258,34 @@ class UpdateParentTag(ImportAction):
 
     def __str__(self) -> str:
         taxonomy_tag = self._get_tag()
+
+        if taxonomy_tag.external_id:
+            prefix_str = _("Update the parent of tag (external_id={external_id})").format(
+                external_id=taxonomy_tag.external_id
+            )
+        else:
+            prefix_str = ""
+            prefix_str = _("Update the parent of tag (value={value})").format(
```

Comment:
Instead of these big `if` statements, I think it would be a much nicer approach to do this:

```
description_str = _("Update the parent of {tag} from parent {old_parent} to {new_parent}").format(
    tag=taxonomy_tag,
    old_parent=taxonomy_tag.parent,
    new_parent=self.tag.parent_id,
)
```

And just make sure that both `Tag` models have a __str__ like:

```
def __str__(self):
    if self.external_id:
        return f'Tag<"{self.value}" ({self.external_id})>'
    return f'Tag<"{self.value}">'
```

What do you think?

Reply:
Yes, is cleaner. I updated that here https://github.com/openedx/openedx-learning/pull/135/commits/6181f7c06f6cbcbcdcb3649f96e7ebc0a93cfba3